build/updater.sh

```diff
@@ -2,6 +2,7 @@
 cd query-builder
 git pull
 rm -rf node_modules
+npm install npm@6.14.8
```

Comment:
I think this should say
```suggestion
npm install -g npm@6.14.8
```
to have the desired effect.